teams/wg-traits.toml

```diff
@@ -1,3 +1,7 @@
+# Note: this has effectively been replaced by the types team
+# This team remains until we figure out what to do with archived teams
```

Comment:
nit: the issue is that we can't archive teams (we already have an archive folder). The issue is that archiving doesn't work if a person in the Team repo is *only* on an archived team. Can we make that explicit here so that no one is tempted in the future to "fix" this by moving this team to the archive folder?

Reply:
Fixed